tests/test_tag_value_parser.py

```diff
@@ -246,6 +267,14 @@ class TestParser(TestCase):
         'FileComment: <text>Very long file</text>'
         ])
 
+    annotation_str = '\n'.join([
+        'Annotator: Person: Jane Doe()',
+        'AnnotationDate: 2010-01-29T18:30:22Z',
+        'AnnotationComment: <text>Document level annotation</text>',
+        'AnnotationType: OTHER',
+        'SPDXREF: SPDXRef-DOCUMENT'
+    ])
```

Comment:
Can't we extract this to top level and use it in both the lexer and the parser tests?

Reply:
done